a3p-integration/proposals/z:acceptance/stakeBld.test.js

```diff
@@ -1,11 +1,9 @@
 import test from 'ava';
 
-import { QueryClient, setupStakingExtension } from '@cosmjs/stargate'; // fails to import after Endo/init
-
-import '@endo/init';
+import { QueryClient, setupStakingExtension } from '@cosmjs/stargate';
+import { Tendermint34Client } from '@cosmjs/tendermint-rpc';
```

Comment:
I see these are what necessitated the patches. We want to avoid patches so we need to make it clear how to remove them. We can't comment in package.json so please document here:

```suggestion
// XXX the @cosmjs packages load `axios` and `protobufjs`  versions that are incompatible with Endo (which Ava has already inited)
// UNTIL https://github.com/cosmology-tech/telescope/issues/692
import { QueryClient, setupStakingExtension } from '@cosmjs/stargate';
import { Tendermint34Client } from '@cosmjs/tendermint-rpc';
```

Reply:
Resolved in https://github.com/Agoric/agoric-sdk/commit/ef6724269607e89a03c72dc3d330cad9875bd491